legal-api/src/legal_api/resources/v1/business/business_filings.py

```diff
@@ -81,7 +81,7 @@ def get(identifier, filing_id=None):  # pylint: disable=too-many-return-statemen
             if not rv.storage:
                 return jsonify({'message': f'{identifier} no filings found'}), HTTPStatus.NOT_FOUND
             if str(request.accept_mimetypes) == 'application/pdf' and filing_id:
-                if rv.filing_type == 'incorporationApplication':
+                if rv.filing_type in ['amalgamationApplication', 'incorporationApplication']:
```

Comment:
unfortunately we are still using v1 resources to get documents in entity-emailer